contents/spotlight/startup-documenso.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "How Documenso got 5,000 stars and grew an active GitHub community"
+date: 2023-09-12
+author: ["joe-martin"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/startup_bugprove.png
+featuredImageType: full
+category: Startups
+---
+
+> Each month, we speak to a team enrolled in [PostHog for Startups](/startups) about the way they work. This time, we spoke to Documenso about how they’ve grown their GitHub community. 
+
+Documenso, the open source alternative to Docusign, launched in November 2022 but has grown rapidly. The project already has over 5,000 GitHub stars at the time of writing and the team track their Star progress closely, viewing it as a vital community metric.
+
+“We track Star, Forks and Contributors on a weekly basis,” says Florent Merian from Documenso. “Stars are mainly useful to us as a social proof. They show that users trust us, and encourage others to do the same — but they also inspire us in the work we do!”
+
+Over the past year Documenso has put a lot of effort into growing its presence on GitHub, and learned several lessons about what does (and doesn’t) work...
+
+![Documenso GitHub Stars](../images/blog/documenso-stars.png)
+<Caption>Documenso has adopted various tactics to create a vibrant GitHub following</Caption>
+
+### 1. Start small, stay transparent
+
+Every journey starts with a single step, and for Documenso that was encouraging the team and their friends to star the repo. This alone helped them reach their first 50 GitHub stars. 
+
+“Those first 50 stars were mostly from friends and team members,” says Florent. “After that, we turned to the open source community.”
+
+The open source community was an obvious place for the team to focus initially, but what really provoked interest and engagement wasn’t the open source software — it was the open source _culture_.
+
+“We try to be as transparent as possible and share everything publicly, even revenue details,” says Florent. “This helped us engage with users on Twitter and, whenever we saw threads about open source software or the like we were able to get involved and link back to GitHub.”
+
+> **Putting it into practice:** Ask friends and family to star your repo, then widen the request to relevant communities you can reach out to. Closed source teams can find communities unique to their industry, and provoke conversations there. 
+
+### 2. Centralize the community 
+
+A hard-learned lesson for Documenso was that the team started off trying to build a following in Slack, which they came to realize wasn’t the best platform for their users. Eventually, they decided to migrate to Discord as a community hub.
```

Comment:
To clarify: we do like Slack. We just felt the need to have an open, safe place for community members to get in touch, brainstorm ideas, and have fun. It doesn’t mean we won’t ever switch back to Slack. The tools of today aren’t the ones of tomorrow :)